views/account/settings/index.ejs

```diff
@@ -4,7 +4,16 @@
     <%- include('../../partials/header'); %>
     <title>Settings</title>
     <link rel="stylesheet" href="/account/settings/style" />
-    <script src="/account/settings/main" type="module"></script>
+    <!-- <script src="/main.js" type="module"></script> -->
```

Comment:
Why was this changed to line 8?

Reply:
i was using this approach to debug the issue of why this ejs file isn't connecting with the main.js file. i'll change it back for consistency purposes.

Reply:
No worries! I fixed it already